src/App/translations/en.ts

```diff
@@ -10,6 +10,33 @@ const en = {
     invalidLocale: 'Invalid Locale',
     goHome: 'Go Home',
   },
+  metadata: {
+    default: {
+      title: 'FreeBoardGame.org',
+      description:
+        'Play board games in your browser for free. Compete against your online friends or play locally. Free and open-source software project.',
+    },
+    about: {
+      title: 'About Us',
+      description: 'About FreeBoardGame.org, a free and open-source software project.',
+    },
+    index: {
+      title: 'Play Free Board Games Online',
+      description:
+        'Play board games in your browser for free. Compete against your online friends or play locally. Free and open-source software project.',
+    },
+    gameTemplate: {
+      title: 'Play Free ${GAME} Online',
+    },
+    chess: {
```

Comment:
this will mix all games in a single file, which is bad. Can move these to a translation file  inside the game folder?